docs/teams-api.mdx

```diff
@@ -13,11 +13,12 @@ All team endpoints leverage the [Team object](teams-model)
 
 ### Parameters
 
-| Type   | Key            | Value                                                       |
-| ------ | -------------- | ----------------------------------------------------------- |
-| Header | X-Effx-Api-Key | Your API key                                                |
-| Query  | limit          | Integer. How many records to retrieve per request           |
-| Query  | offset         | Integer. The number of records to skip. Use for pagination. |
+| Type   | Key            | Value                                                                                                   |
+| ------ | -------------- | --------------------------------------------------------------------------------------------------------|
+| Header | X-Effx-Api-Key | Your API key                                                                                            |
+| Query  | limit          | Integer. How many records to retrieve per request                                                       |
+| Query  | offset         | Integer. The number of records to skip. Use for pagination.                                             |
```

Comment:
should it be used for pagination or use? not sure

Reply:
i think either is probably fine - this line didn't change so maybe we just leave it haha